flutter-studio/src/io/flutter/profiler/FlutterStudioMonitorStageView.java

```diff
@@ -919,4 +922,76 @@ void getObject(DefaultMutableTreeNode parent, String objectRef) {
       }
     });
   }
+
```

Comment:
I need this as well to show a nice hover for the table of frequent rebuilds. Can you move this class into the view directory and make it public?

Reply:
I'll move this over in the next PR I send you today.